src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The `href` attribute in the link tag should point to a CSS file, not a SCSS file. Please ensure that the path is correct and points to a compiled CSS file.